ci/auto-pr/create_pull_requests

```diff
@@ -39,6 +46,8 @@ function cherry_pick_and_create_pull_request () {
 }
 
 function create_issue () {
+  local branch=$1
```

Comment:
This is a bit unrelated bug fix that the global variable is used unexpectedly...